product_multi_barcode_constraint_per_company/tests/test_product_barcode_constraint.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import types
+
+from psycopg2 import IntegrityError
+
+from odoo.tests.common import tagged
+from odoo.tools.misc import mute_logger
+
+from odoo.addons.product_barcode_constraint_per_company.tests.test_module import (
+    TestModule,
+)
+
+from .. import void
+
+
+@tagged("post_install", "-at_install")
+class TestProductBarcodeConstraintInherit(TestModule):
+    def test_void(self):
+        """Verify the type of the result of the void method call."""
+        result = void(None)
+        self.assertIsInstance(
+            result,
+            types.FunctionType,
+            msg="Expecting a FunctionType result, but got {result_type}".format(
+                result_type=type(result)
+            ),
+        )
+
+    def test_create_same_company(self):
+        """Verifying the Existence of a Product with the Same Barcode within a Single Company"""
+        super().test_create_same_company()
```

Comment:
You deactivate parent function. Why do you use `super` here?

Reply:
Thanks, I'll fix it